src/App.vue

```diff
@@ -671,7 +676,7 @@ export default class App extends Vue {
       // then try to re-route them
       if (this.$route.meta.filingType !== this.getFilingType) {
         switch (this.getFilingType) {
-          case FilingTypes.VOLUNTARY_DISSOLUTION:
```

Comment:
As mentioned in another comment, there is no such filing type. However, the old code worked because VOLUNTARY_DISSOLUTION = `dissolution`.

Two wrongs make a right?